content/source/docs/enterprise/private/config.html.md

```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 In all examples below, be sure to replace "`<TFE HOSTNAME>`" with the hostname
 of your Private Terraform Enterprise instance.
 
-Navigate to `https://<TFE HOSTNAME>/admin/bootstrap` in your browser. You will
-be presented with a status page that looks like this:
+Navigate to `https://<TFE HOSTNAME>:8800/` in your browser. You will
+be presented with the installer's dashboard that looks like this:
 
-![Bootstrap Status Page](./assets/bootstrap-status.png)
+![PTFE Installer Dashboard](./assets/post-install-repl-console.png)
 
 ### Creating an Administrator
 
-After clicking on "Begin Installation" you will be brought to a page called
-"Create Administrator". Enter the requested information to create the first
-Administrator account. You will be able to create additional administrators
-later.
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
+You will be able to create additional administrators once you login.
+
+![New admin dialog](./assets/create-an-account.png)
 
 ### Creating an Organization
 
 The next step will create the first Organization.
 
 ![Create a new organization](./assets/create-organization.png)
 
-If you skip this step, you will be prompted the create the first organization
-when you first log in.
-
-After this is done, and showing the `Create a new Workspace` page, click back to the previous tab/browser window and continue.
-
-### Configure SMTP (optional)
-
-Terraform Enterprise sends emails occasionally for features like password reset
-and to alert you when a new administrator is added. Fill out the details for
-your SMTP server.  You may use an external provider such as Amazon SES or
-Mailgun.
-
-When you have added your configuration press "Save and Continue," or skip to
-the next step.
-
-### Installation Completion
-
-At this point the installation is complete! You will be presented with this
-page:
-
-![Installation complete page](./assets/installation-complete.png)
-
-There are two tertiary links on the bottom of the page:
-
- * **Configure GitHub** - [**DO NOT USE**] This configuration mechanism is
-   deprecated and will be removed in a future version of Private Terraform
-   Enterprise.
- * **Configure Twilio** - This will set up Twilio for SMS-based multi-factor
-   authentication see the next section for details.
-
-Click **Create a Terraform workspace** when you are ready to continue.
-
-### Configure Twilio (optional)
-
-Terraform Enterprise uses Twilio to send SMS if a user selects SMS as their multi-factor
-authentication option. If Twilio is not configured they must use the Google
-Authenticator app on their smartphone instead.
-
-When you have added your configuration press "Save and Continue," or skip to the
-next step.
+After this is done, you can either continue with the creation of a new workspace,
+choose to configure other aspects of PTFE, or add more users.
```

Comment:
I might change this to explicitly include a mention of "configure login settings such as SAML or Twilio", with links to the relevant configuration pages. (If we don't have existing details for Twilio, we'll need them. :)